test/pull/out/bar/transformation/keboola.python-transformation-v2/python-transformation-shared-code/config.json

```diff
@@ -1,3 +1,9 @@
 {
-  "parameters": {}
+  "parameters": {},
+  "shared_code_id": "...",
```

Comment:
Chybajuce polozky su uz tu.
Dal som to do tohto PR, aby to nerobilo chaos v dalsich PR.